### ISSUE TITLE
Accessibility: account, provisioning/create, explorer

### ISSUE THREAD

**Issue by lvuch** [rancher/dashboard#4390]
Master: https://github.com/rancher/dashboard/issues/4324
4.
### Shared issues

- [Repeats in other pages] Arrows not in tabbing order ![](RackMultipart20211014-4-19dhorn_html_9cf8413f26bee7bc.png) **image referenced**: the arrows in tables which order results cannot be accessed with the keyboard. The arrow icons are also missing labels. Consider using a tooltip for these scenarios since not only visually impaired users will be benefited from them. The element should also be focusable and have an appropriate :focus style.
- [Repeats in other pages] Dash color contrast fails. Consider using the same color as used in text or adding a &quot;aria-hidden&quot; to it.
- [Repeats in other pages] Active label fails contrast. The table has 3 different states where the background changes colors. It is important that the color of the text has a good contrast (4.5:1) compared against the 3 backgrounds. Also, consider the same check for Dark-theme. ![](RackMultipart20211014-4-19dhorn_html_b4a6bf3f4ae36a39.png) **image referenced**
- [Repeats in other pages] Form elements should have a visible text label ![](RackMultipart20211014-4-19dhorn_html_752c3d5d28141524.png) **image referenced**
- ON DIALOG: CHANGE PASSWORD: The blue outlined button to cancel the action fails color contrast heavily when the button is focused: ![](RackMultipart20211014-4-19dhorn_html_a0786ac29ee29eba.png) **image referenced** . When the button isn&#39;t focused it also fails color contrast criterias: ![](RackMultipart20211014-4-19dhorn_html_37e5706f1fb17236.png) **image referenced** ![](RackMultipart20211014-4-19dhorn_html_9126a8461af4272b.png) **image referenced**
- ON DIALOG: CHANGE PASSWORD: The red button fails color contrast ![](RackMultipart20211014-4-19dhorn_html_58c5b071ca8f22d5.png) **image referenced** ![](RackMultipart20211014-4-19dhorn_html_7385af3b5b506042.png) **image referenced**
- [Repeats in other pages] Table header can&#39;t be empty:

![](RackMultipart20211014-4-19dhorn_html_e4d57472224cc129.png) **image referenced**

5.
### Shared issues

- Card elements not in tabbing order (can&#39;t be accessed with the keyboard)

![](RackMultipart20211014-4-19dhorn_html_8cb4acbe1fefd2f8.png) **image referenced**

- Labels missing in logo images.
- Version text in back left of screen. Color contrast fails

![](RackMultipart20211014-4-19dhorn_html_36f76a2a251d66be.png) **image referenced**

### Dark theme issues

- Color contrast fails in side nav elements

![](RackMultipart20211014-4-19dhorn_html_353f5eadcd369061.png) **image referenced**

6.
### Shared issues

- Missing focus style on icons in toolbar
![](RackMultipart20211014-4-19dhorn_html_def97c2d7ddac2c8.png) **image referenced**

- ![](RackMultipart20211014-4-19dhorn_html_e644f8d33d8bf1ad.png) **image referenced** Clickable elements must be focusable and should have interactive semantics. [More on this.](https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_WCAG/Keyboard#clickable_elements_must_be_focusable_and_should_have_interactive_semantics)

- Color contrast fails for the muted text showing the used resources: ![](RackMultipart20211014-4-19dhorn_html_653279bc8e83e2da.png) **image referenced**
